posthog/hogql/database/schema/revenue_analytics.py

```diff
@@ -34,7 +34,7 @@ def join_with_persons_revenue_analytics_table(
             constraint=ast.JoinConstraint(
                 expr=ast.CompareOperation(
                     op=ast.CompareOperationOp.Eq,
-                    left=ast.Field(chain=[join_to_add.from_table, "id"]),
```

Comment:
TODO: This needs to be dynamic, it still needs to use `id` when pointing at the `persons` table directly rather than through `events`